### DISCUSSION TITLE
Update Button Grayed out for Online Only meetings

### DISCUSSION THREAD

**Discussion by michaelvenzor** [code4recovery/12-step-meeting-list#1558] [Q&A] [answered]
When I edit any online-only meeting (“Can I currently attend this meeting in person?” is set to NO), the Update button is not clickable; it's grayed out.

And nothing I do or change makes it clickable again besides changing the meeting to hybrid (“Can I currently attend this meeting in person?” is set to YES).

But even if I do that, when I change it back to NO, the Update button grays out again.

I've searched for a similar issue and found one other post, but no answers help as of yet. Is anyone else experiencing this issue?

My URL is https://brooklynintergroup.org/brooklyn/meetings/

**Comment by joshreisner** [comment]
hm, we're not seeing that on our demo site- i'm able to create and edit an online-only meeting: https://demo.code4recovery.org/meetings/online-only-meeting

what zoom address and what physical address is entered on the meeting you're trying to edit? (if you can add a screenshot that could help but know that email attachments don't work with github - you need to come to this discussion thread and upload it here)

**Comment by michaelvenzor** [comment]
Here's an example. When I click on Edit for this meeting, the Update button is grayed out and any changes I make don't make it clickable, besides changing the meeting to Hybrid by allowing attendance in person.

URL: https://zoom.us/j/376702512
Location: St. Francis Xavier School, 752 President St, Brooklyn, NY 11215, USA

I don't know what you mean about coming to the discussion thread to upload screenshots. How do I exit github but still stay in this discussion thread? Anyway, here are links to the two screenshots:
https://brooklynintergroup.org/brooklyn/wp-content/uploads/2024/11/Screenshot-2024-11-01-at-11.41.49 AM.png
https://brooklynintergroup.org/brooklyn/wp-content/uploads/2024/11/Screenshot-2024-11-01-at-11.42.16 AM.png

**Comment by joshreisner** [comment]
github allows people to reply by email, and often when we request screenshots people attach them to the email, which just shows up in the thread as text

screenshots look great, thanks. not seeing any issues

my guess is there is some kind of javascript error happening (maybe from another plugin?) which is preventing our plugin from working. would you be able to look in your browser's [javascript console](https://www.digitalocean.com/community/tutorials/how-to-use-the-javascript-developer-console#working-with-the-console-in-a-browser) and let us know if there are any errors there? should look something like this:

<img width="1728" alt="Screenshot 2024-11-01 at 9 38 07 AM" src="https://github.com/user-attachments/assets/ffbeee03-a5a1-4964-bf13-f36d6149e0fc">

not all errors are blockers though, since there are a few above even though our update button is still blue

**Comment by michaelvenzor** [comment, marked as answer]
Thanks, that got me looking closer. Got it. The issue was that the "location" I was using for all Online Meetings was "Flatbush, Brooklyn, NY, USA" and it was being flagged as a not approximate address. I changed them all to "Brooklyn, NY, USA", which is approxmiate, and it works now.
